Description: Remove alloc_trait feature
 This feature is unused by reverse dependencies and it doesn't build with recent versions of Rust.
Author: James McCoy <jamessan@debian.org>
Bug: https://github.com/tikv/jemallocator/issues/38
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: tikv-jemallocator/Cargo.toml
===================================================================
--- tikv-jemallocator.orig/Cargo.toml
+++ tikv-jemallocator/Cargo.toml
@@ -63,7 +63,6 @@ version = "1"
 version = "0.5.0"
 
 [features]
-alloc_trait = []
 background_threads = ["tikv-jemalloc-sys/background_threads"]
 background_threads_runtime_support = ["tikv-jemalloc-sys/background_threads_runtime_support"]
 debug = ["tikv-jemalloc-sys/debug"]
